pkg/services/dashboards/database/database_dashboard_public_test.go

```diff
@@ -0,0 +1,233 @@
+//go:build integration
```

Comment:
we're not using integration tags anymore, these tests are not being run in the CI, instead we use naming `TestIntegration***()`